.github/workflows/tdd.yml

```diff
@@ -49,12 +49,11 @@ jobs:
           # test for complexity. The GitHub editor is 127 chars wide
           flake8 service tests --count --max-complexity=10 --max-line-length=127 --statistics
           # Run pylint to catch other PEP8 errors
-          pylint service tests --max-line-length=127
+          pylint service tests --max-line-length=127 --fail-under=9
 
       - name: Run unit tests with green
         run: |
             export FLASK_APP=service:app
-            flask db-create
```

Comment:
This seems like a nice thing to have. Unfortunately, I'm now getting a compiler error if I include this line

The "flask db-create" endpoint code is still in the codebase, so I'm not sure how adding Flask-RestX breaks this